Failed to execute Antrun job for module mnemonic-nvml-vmem-service

The project building shows the following error with openJDK 11

```
[INFO] --- maven-antrun-plugin:1.8:run (build-native) @ mnemonic-nvml-vmem-service ---
[INFO] Executing tasks

main:
     [echo] Java Home: /usr/lib/jvm/java-11-openjdk-amd64
     [exec] -- Configuring incomplete, errors occurred!
     [exec] See also "/home/gary/ws/mnemonic/mnemonic-memory-services/mnemonic-nvml-vmem-service/src/main/native/build/CMakeFiles/CMakeOutput.log".
     [exec] See also "/home/gary/ws/mnemonic/mnemonic-memory-services/mnemonic-nvml-vmem-service/src/main/native/build/CMakeFiles/CMakeError.log".
     [exec] CMake Error at CMakeLists.txt:43 (message):
     [exec]   not found vmem library
     [exec] 
     [exec] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  5.071 s
[INFO] Finished at: 2020-07-18T13:39:46-07:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (build-native) on project mnemonic-nvml-vmem-service: An Ant BuildException has occured: exec returned: 1
[ERROR] around Ant part ...<exec failonerror="true" dir="src/main/native/build" executable="cmake">... @ 6:75 in /home/gary/ws/mnemonic/mnemonic-memory-services/mnemonic-nvml-vmem-service/target/antrun/build-main.xml
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

```

--- a/mnemonic-computing-services/mnemonic-utilities-service/src/main/native/CMakeLists.txt
+++ b/mnemonic-computing-services/mnemonic-utilities-service/src/main/native/CMakeLists.txt
@@ -41,6 +41,7 @@ add_library(utilitiescomputing SHARED common.c
   org_apache_mnemonic_service_computing_internal_PrintServiceImpl.c
   org_apache_mnemonic_service_computing_internal_VectorizationServiceImpl.c)
 target_include_directories(utilitiescomputing PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_link_libraries(utilitiescomputing ${CMAKE_THREAD_LIBS_INIT})
 
 include (InstallRequiredSystemLibraries)
 set (CPACK_RESOURCE_FILE_LICENSE

--- a/mnemonic-memory-services/mnemonic-nvml-pmem-service/src/main/native/CMakeLists.txt
+++ b/mnemonic-memory-services/mnemonic-nvml-pmem-service/src/main/native/CMakeLists.txt
@@ -50,7 +50,7 @@ endif (NOT LIBPMEMOBJ_LIBRARIES)
 
 add_library(pmemallocator SHARED common.c org_apache_mnemonic_service_memory_internal_PMemServiceImpl.c)
 target_include_directories(pmemallocator PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_link_libraries(pmemallocator ${LIBPMEMOBJ_LIBRARIES} ${LIBPMEM_LIBRARIES})
+target_link_libraries(pmemallocator ${LIBPMEMOBJ_LIBRARIES} ${LIBPMEM_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
 
 include (InstallRequiredSystemLibraries)
 set (CPACK_RESOURCE_FILE_LICENSE

--- a/mnemonic-memory-services/mnemonic-nvml-vmem-service/src/main/native/CMakeLists.txt
+++ b/mnemonic-memory-services/mnemonic-nvml-vmem-service/src/main/native/CMakeLists.txt
@@ -45,7 +45,7 @@ endif (NOT LIBVMEM_LIBRARIES)
 
 add_library(vmemallocator SHARED common.c org_apache_mnemonic_service_memory_internal_VMemServiceImpl.c)
 target_include_directories(vmemallocator PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_link_libraries(vmemallocator ${LIBVMEM_LIBRARIES})
+target_link_libraries(vmemallocator ${LIBVMEM_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
 
 include (InstallRequiredSystemLibraries)
 set (CPACK_RESOURCE_FILE_LICENSE

--- a/mnemonic-memory-services/mnemonic-pmalloc-service/src/main/native/CMakeLists.txt
+++ b/mnemonic-memory-services/mnemonic-pmalloc-service/src/main/native/CMakeLists.txt
@@ -47,7 +47,7 @@ endif (NOT LIBPMALLOC_LIBRARIES)
 
 add_library(pmallocallocator SHARED common.c org_apache_mnemonic_service_memory_internal_PMallocServiceImpl.c)
 target_include_directories(pmallocallocator PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_link_libraries(pmallocallocator ${LIBPMALLOC_LIBRARIES})
+target_link_libraries(pmallocallocator ${LIBPMALLOC_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
 
 include (InstallRequiredSystemLibraries)
 set (CPACK_RESOURCE_FILE_LICENSE

--- a/mnemonic-memory-services/mnemonic-pmdk-pmem-service/src/main/native/CMakeLists.txt
+++ b/mnemonic-memory-services/mnemonic-pmdk-pmem-service/src/main/native/CMakeLists.txt
@@ -50,7 +50,7 @@ endif (NOT LIBPMEMOBJ_LIBRARIES)
 
 add_library(pmdkpmemallocator SHARED common.c org_apache_mnemonic_service_memory_internal_PMDKPMemServiceImpl.c)
 target_include_directories(pmdkpmemallocator PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_link_libraries(pmdkpmemallocator ${LIBPMEMOBJ_LIBRARIES} ${LIBPMEM_LIBRARIES})
+target_link_libraries(pmdkpmemallocator ${LIBPMEMOBJ_LIBRARIES} ${LIBPMEM_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
 
 include (InstallRequiredSystemLibraries)
 set (CPACK_RESOURCE_FILE_LICENSE

--- a/mnemonic-memory-services/mnemonic-pmdk-vmem-service/src/main/native/CMakeLists.txt
+++ b/mnemonic-memory-services/mnemonic-pmdk-vmem-service/src/main/native/CMakeLists.txt
@@ -45,7 +45,7 @@ endif (NOT LIBVMEM_LIBRARIES)
 
 add_library(pmdkvmemallocator SHARED common.c org_apache_mnemonic_service_memory_internal_PMDKVMemServiceImpl.c)
 target_include_directories(pmdkvmemallocator PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_link_libraries(pmdkvmemallocator ${LIBVMEM_LIBRARIES})
+target_link_libraries(pmdkvmemallocator ${LIBVMEM_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
 
 include (InstallRequiredSystemLibraries)
 set (CPACK_RESOURCE_FILE_LICENSE

--- a/mnemonic-memory-services/mnemonic-sys-vmem-service/src/main/native/CMakeLists.txt
+++ b/mnemonic-memory-services/mnemonic-sys-vmem-service/src/main/native/CMakeLists.txt
@@ -40,6 +40,7 @@ include_directories(${CMAKE_THREAD_LIBS_INIT})
 
 add_library(sysvmemallocator SHARED common.c org_apache_mnemonic_service_memory_internal_SysVMemServiceImpl.c)
 target_include_directories(sysvmemallocator PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_link_libraries(sysvmemallocator ${CMAKE_THREAD_LIBS_INIT})
 
 include (InstallRequiredSystemLibraries)
 set (CPACK_RESOURCE_FILE_LICENSE

--- a/pom.xml
+++ b/pom.xml
@@ -369,7 +369,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-antrun-plugin</artifactId>
-          <version>1.8</version>
+          <version>3.0.0</version>
         </plugin>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>

